### ISSUE TITLE
[Q] Calculate metrics from charts

### ISSUE THREAD

**Issue by pablo2909** [wandb/wandb#5709]
Hi,

I have five training loss curves, that I grouped into a mean curve and its standard deviation. I would like to compute the average of the standard deviation across the training steps. Can this be done ? 

Thank you

**Comment by luisbergua** [comment]
Hi @pablo2909, thanks for your question! You can add expressions to your plot as explained [here](https://docs.wandb.ai/guides/app/features/panels/line-plot#settings), would this be useful? If not, could you please share a link to your Workspace so I can have a look at this?

**Comment by pablo2909** [comment]
Hi @luisbergua , thank you for getting back to me. I can invite you to look at one of the plots. Would that allow you to help me ? If yes, which address should I use ?

**Comment by luisbergua** [comment]
That would be useful @pablo2909! Would you mind sharing the page url and the name of the plot so I can see it?

**Comment by pablo2909** [comment]
Here it is: https://api.wandb.ai/links/pablo2909/e2ucahpb

**Comment by luisbergua** [comment]
Thanks @pablo2909! So the ideal behavior here would be to have like a "summary" when grouping so you can easily see information about stddev, right? This isn't possible at this moment so I'll create a new feature request. It'd be great if you could also share some details on your sue case so we can have a better context here!

**Comment by pablo2909** [comment]
Yes, that would be neat. 

More broadly, it would be interesting to be able to do more "complex" operations on a graph, or a group of graph. For example, computing the mean of the graph across time, its variance, the correlation between graphs etc. 

Thank you for the help, I'll close the issue.
